apps/nuxt-ssr/pages/_schema/ssr-catalogue.vue

```diff
@@ -2,8 +2,21 @@
 <div>
   <Menu :menu="menu"/>
   <div class="container pt-3">
-    <h1>Data Catalogue for schema: {{$route.params.schema}}</h1>
-        <h2>Collected data</h2>
+    <h1>
```

Comment:
This one we don't really need in ssr? Or is your idea to move whole catalogue into ssr?

Reply:
yes , move the whole catalogue